packages/console/src/scss/_colors.scss

```diff
@@ -59,6 +59,7 @@
   --color-neutral-variant-99: #fffbff;
   --color-overlay-8: rgba(25, 28, 29, 8%);
   --color-overlay-12: rgba(25, 28, 29, 12%);
+  --color-border: #c4c7c780;
```

Comment:
`--color-border` doesn't belong here, and the value should be `#c4c7c7`, which is `--color-neutral-80`

<img width="484" alt="image" src="https://user-images.githubusercontent.com/12833674/162139441-c3e1d49a-a9be-482c-b42c-be0951ab7cf5.png">

Reply:
updated, PTAL